css/styles.css

```
@@ -393,40 +685,96 @@ body {
   height: 2px;
   background-color: #000000;
 }
-/* Псевдокласс - :focus   !!!НЕ РАБОТАЕТ!!!  */
+
+/* Псевдокласс - :focus*/
 .main-navigation-link:focus {
   border: 2px solid #af4fff;
-  box-sizing: border-box;
-  border-radius: 25px;
+  position: relative;
+  z-index: 1;
+}
+.main-navigation-item:focus-within .subcatalog-list {
+  visibility: visible;
 }
 /* Псевдокласс - :active  */
 .main-navigation-link:active {
   border: 2px solid transparent;
-  box-sizing: border-box;
-  border-radius: 25px;
   opacity: 0.3;
 }
 .main-navigation-link:active .catalog-title::after {
   opacity: 0.3;
   background: url(../img/icon/catalog-minus.svg) no-repeat center center;
 }
+/* Псевдокласс - :hover  */
+.subcatalog-link:hover {
+  opacity: 0.6;
+}
+/* Псевдокласс - :focus*/
+.subcatalog-link:focus {
+  border: 2px solid #af4fff;
+}
```

```suggestion
.subcatalog-link:hover {
  opacity: 0.6;
}

/* Псевдокласс - :focus*/
.subcatalog-link:focus {
  border: 2px solid #af4fff;
}
```
было бы классно, если бы ты так добавила везде. обычно пишут так)